src/libcollections/slice.rs

```diff
@@ -2127,22 +2032,6 @@ mod tests {
     }
 
     #[test]
-    fn test_partition() {
```

Comment:
can you change these to test the new partition function in iter (possibly moving them if you think there's a better spot)? i neglected to add tests for it...